js/web/lib/wasm/jsep/init.ts

```diff
@@ -54,6 +54,7 @@ class TensorViewImpl implements TensorView {
 }
 
 class ComputeContextImpl implements ComputeContext {
+  readonly adapterInfo: AdapterInfo;
```

Comment:
use a more general name, like `features` for the feature manager. Code that using it can be something like
```
if (await context.features.useGroupedConvVectorization()) {
   ...
}

Reply:
As I replied above, it's not my intension to control all such features at a centric place now. ComputeContext just passes the info that may be helpful for kernels, but it has no idea how these info would be used.